packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -1194,19 +1194,16 @@ enum ResizeImagePolicy {
 /// Instructs Flutter to decode the image at the specified dimensions
 /// instead of at its native size.
 ///
-/// The [width] and [height] parameters refer to logical pixels.
-///
-/// Logical pixels have roughly the same visual size across devices, whereas
-/// physical pixels correspond to actual device hardware.
```

Comment:
This sentence (which was [written by me](https://github.com/flutter/flutter/pull/154212#discussion_r1735434443)) is unnecessarily confusing, since an unconstrained `ResizeImage`'s scaling to match logical pixels means that by default, it doesn't match the resolution of the actual device hardware.